sssd_test_framework/hosts/client.py

```diff
@@ -41,7 +41,7 @@ def features(self) -> dict[str, bool]:
             """
             set -ex
 
-            [ -f "/usr/lib64/sssd/libsss_files.so" ] && echo "files-provider"
+            [ -f "/usr/lib64/sssd/libsss_files.so" ] && echo "files-provider" || exit 0
```

Comment:
This will exit the existing session without breaking the tests. But will it detect when the feature is missing?

Reply:
Yes, the following lines of code create a features dict, enabling the feature based on the string echo'd back from this line. If no string is echo'd back then the feature is disabled.

Reply:
Updated.